ROADMAP.md

```diff
@@ -21,17 +21,17 @@ This roadmap represents some of priorities for us over the next couple months. I
 - [x] Choose a chat platform for maintainers and contributors, and post notices in various existing places (#hubot on freenode, github/hubot on Gitter). Slack is the obvious choice here. [Join us on Slack](https://hubot-slackin.herokuapp.com/).
 - [x] Add a code of conduct based on http://contributor-covenant.org/ and processes to enforce it in all official spaces. ([#1334](https://github.com/github/hubot/pull/1334))
 - [ ] Publish weekly community updates (blog, newsletter, etc) which highlight recent and upcoming changes, give shoutouts to contributors / maintainers, and maybe mention interesting uses of Hubot
-- [ ] Create Hubot Evolution—inspired by [Swift Evolution](https://github.com/apple/swift-evolution)—for proposing user-visible enhancements. This roadmap will be moved there and all future proposals will follow the process laid out in that repository. ([hubotio/evolution#1](https://github.com/hubotio/evolution/pull/1))
+- [x] Create Hubot Evolution—inspired by [Swift Evolution](https://github.com/apple/swift-evolution)—for proposing user-visible enhancements. ([hubotio/evolution#1](https://github.com/hubotio/evolution/pull/1))
```

Comment:
@technicalpickles @mose @gr2m initially I was thinking we'd have all the other items on this list done by the time the evolution process was established, and then remove this roadmap.  But as we've seen it's hard to clean up old isuses/PRs without giving people a place to discuss features.

What do you think about leaving this list here until we get all the items checked off?